diff --git a/httpx/_config.py b/httpx/_config.py
index dbd2b46..467a6c9 100644
--- a/httpx/_config.py
+++ b/httpx/_config.py
@@ -39,10 +39,9 @@ def create_ssl_context(
             # Default case...
             ctx = ssl.create_default_context(cafile=certifi.where())
     elif verify is False:
-        ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
-        ssl_context.check_hostname = False
-        ssl_context.verify_mode = ssl.CERT_NONE
-        return ssl_context
+        ctx = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+        ctx.check_hostname = False
+        ctx.verify_mode = ssl.CERT_NONE
     elif isinstance(verify, str):  # pragma: nocover
         message = (
             "`verify=<str>` is deprecated. "
diff --git a/tests/test_config.py b/tests/test_config.py
index 22abd4c..f149295 100644
--- a/tests/test_config.py
+++ b/tests/test_config.py
@@ -1,6 +1,7 @@
 import ssl
 import typing
 from pathlib import Path
+from unittest.mock import patch
 
 import certifi
 import pytest
@@ -79,6 +80,22 @@ def test_load_ssl_config_no_verify():
     assert context.check_hostname is False
 
 
+def test_load_ssl_config_no_verify_with_cert():
+    with patch("ssl.SSLContext") as ssl_context_cls:
+        context = ssl_context_cls.return_value
+        with pytest.warns(DeprecationWarning):
+            result = httpx.create_ssl_context(
+                verify=False, cert=("client-cert.pem", "client-key.pem")
+            )
+
+    assert result is context
+    assert context.verify_mode == ssl.VerifyMode.CERT_NONE
+    assert context.check_hostname is False
+    context.load_cert_chain.assert_called_once_with(
+        "client-cert.pem", "client-key.pem"
+    )
+
+
 def test_SSLContext_with_get_request(server, cert_pem_file):
     context = httpx.create_ssl_context()
     context.load_verify_locations(cert_pem_file)
